### PR TITLE
Add Data File parser

### PR DESCRIPTION
**Pull request by 0ddlyoko**
This PR will add the parser of data files

Minecraft data are taken from https://github.com/PrismarineJS/minecraft-data

For now, only blocks are imported. We'll import other data later

### CODE REVIEW

**Comment by 0ddlyoko** [comment]
Done & ready for a review
Only versions >= 13.0 works, we will handle versions < 13.0 later

File: classes/utils/Utils.py [hunk 1]
```diff
@@ -49,23 +60,25 @@ def __gt__(self, other):
...
-    v1_14_2 = (1, 14, 2, 485)
-    v1_14_3 = (1, 14, 3, 490)
-    v1_14_4 = (1, 14, 4, 498)
-    v1_15_0 = (1, 15, 0, 573)
-    v1_15_1 = (1, 15, 1, 575)
-    v1_15_2 = (1, 15, 2, 578)
-    v1_16_0 = (1, 16, 0, 735)
-    v1_16_1 = (1, 16, 1, 736)
-    v1_16_2 = (1, 16, 2, 751)
+    v1_12_0 = (1, 12, 0, 335, '1.12')
```
Comment (hydrostaticcog) [inline classes/utils/Utils.py:63]: loading all of the versions takes time/its not really necessary.

File: main.py [hunk 2]
```diff
@@ -328,6 +328,35 @@ def _launch_server():
...
 
+    print("Loading data files, please wait ...")
+    # TODO Import it in another class
+    print("Data files loaded !")
+    print("Importing Minecraft data, please wait ...")
+    from classes.blocks.Materials import Material
+    from classes.utils.Parser import Parser
+    from classes.utils.Utils import Version
+    versions_string = [
+        '1.13',
```
Comment (hydrostaticcog) [inline main.py:339]: we might want to hardcode the logging of versions this is loading at some point, to avoid logging incorrect version numbers.

**Review by hydrostaticcog — COMMENTED** [review]
Looks good. One thing: this currently loads multiple versions of data when we only need/are testing 1.15.2. Also, the logging in main.py has hardcoded data logs. We should change this to only log what version of data that we are loading/what we need to load.

**Comment by 0ddlyoko** [comment]
> Looks good. One thing: this currently loads multiple versions of data when we only need/are testing 1.15.2. Also, the logging in main.py has hardcoded data logs. We should change this to only log what version of data that we are loading/what we need to load.

Thanks for the review, it's strange because I'm pretty sure I have updated the code.

I think I have updated the code and I haven't pushed it, fool that I am

**Comment by hydrostaticcog** [comment]
Developer Left Project/PR left with conflicts

**Comment by Geolykt** [comment]
I have looked at the PR again (given it's relative importance) and if the merge conflicts are naively resolved it still will not start up at all due to a circulatory import. I will look at what I can do to resolve that however

**Pull request closed without merging**